client/client-bundle.js

```diff
@@ -900,7 +900,7 @@ function parse(html, doc) {
   }
 
   // wrap map
```

Comment:
Usually, I would prefer the plug-in and the other outputs that get deployed to sit in a dedicated folder (e.g. called `dist`), but I don't know too well how the plugin integrates with Camunda Modeler, yet, so I won't move it around in this PR.

Reply:
Ok, I found out how I could do it. 

If you want me to, I can easily add these changes as well: 

```diff
Index: package.json
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/package.json b/package.json
--- a/package.json	(revision c1c49131907503134943082321d563f5e10b6b73)
+++ b/package.json	(date 1694520678008)
@@ -4,8 +4,8 @@
   "description": "bpmn-js Rename Technical IDs for the Camunda Modeler",
   "main": "index.js",
   "scripts": {
-    "client": "browserify client/client.js > client/client-bundle.js",
-    "auto-build": "watchify client/client.js -o client/client-bundle.js"
+    "client": "browserify client/client.js > dist/client-bundle.js",
+    "auto-build": "watchify client/client.js -o dist/client-bundle.js"
   },
   "repository": {
     "type": "git",
Index: index.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/index.js b/index.js
--- a/index.js	(revision c1c49131907503134943082321d563f5e10b6b73)
+++ b/index.js	(date 1694520654104)
@@ -3,6 +3,6 @@
 module.exports = {
   name: 'Rename Technical IDs',
   menu: './menu/menu.js',
-  script: './client/client-bundle.js',
+  script: './dist/client-bundle.js',
   style: './style/style.css'
 };
Index: dist/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/dist/.gitignore b/dist/.gitignore
new file mode 100644
--- /dev/null	(date 1694520705976)
+++ b/dist/.gitignore	(date 1694520705976)
@@ -0,0 +1,1 @@
+# file created so that git will track this folder
\ No newline at end of file
diff --git a/client/client-bundle.js b/dist/client-bundle.js
rename from client/client-bundle.js
rename to dist/client-bundle.js
```